+nosnoc/+solver/RelaxationSolver.m

```diff
@@ -146,11 +154,7 @@
 
         function violation = compute_constraint_violation(obj, w, g)
             nlp = obj.nlp;
-            ubw_violation = max(max(w - nlp.w.mpcc_w(0).ub), 0);
-            lbw_violation = max(max(nlp.w.mpcc_w(0).lb - w), 0);
-            ubg_violation = max(max(g - nlp.g.mpcc_g(0).ub), 0);
-            lbg_violation = max(max(nlp.g.mpcc_g(0).lb - g), 0);
-            violation = max([lbg_violation, ubg_violation, lbw_violation, ubw_violation]);
+            violation = max([nlp.g.mpcc_g().violation; nlp.w.mpcc_w().violation]);
```

Comment:
would you mind adding one line of comment explaing this vdx synthax here (what is the output of violation

Reply:
Comment added on violation.